src/lua.rs

```diff
@@ -125,6 +125,18 @@ impl Lua {
         create_lua(lua_mod)
     }
 
+    /// Loads standard the specified set of the standard libraries into an existing Lua state.
+    ///
+    /// Use the [`StdLib`] flags to specifiy the libraries you want to load.
+    ///
+    /// This function is unsafe because it can be used to load the `debug` library which can be used
+    /// to break the safety guarantees provided by rlua.
+    pub unsafe fn load_from_std_lib(&self, lua_mod: StdLib) -> Result<()> {
```

Comment:
Could this be `load_std_lib` and also can there be an `unsafe_load_std_lib` variant that allows debug?

Reply:
`load_std_lib` implies to me that it loads the whole std lib. Maybe `load_from_std_lib` & `unsafe_load_from_std_lib`? Also any preference on if the first should error if debug is in the list or just ignore it?

Reply:
Yeah you're right, I see your point on `load_from_std_lib`, that's fine.

The safe version should *panic* I think if it's asked to load the debug library (that's what new does).

Reply:
Ah, should have looked at that. :+1: 

Changed to `load_from_std_lib` and `unsafe_load_from_std_lib`. (Also squashed the fixups.)